README.md

```diff
@@ -2,0 +1,17 @@
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/troubleshoot-codacy/codacy-test-fork-markdown-test/pullRequest?prid=7586344)
```suggestion

- Identify new Static Analysis issues
```